pkg/apis/core/v1alpha1/bindingphase_test.go

```diff
@@ -52,7 +52,7 @@ func TestBindingStateMarshalJSON(t *testing.T) {
 				t.Errorf("BindingState.MarshalJSON(): %v", err)
 			}
 			if diff := cmp.Diff(tc.want, got); diff != "" {
-				t.Errorf("BindingState.MarshalJSON(): want != got\n %+v", diff)
+				t.Errorf("BindingState.MarshalJSON(): -want, +got\n %+v", diff)
```

Comment:
Should it be `%s` vs. `%+v`?

Reply:
I'd prefer `%s`, but `%+v` is functionally equivalent in this case. Let's align on `%s` next time we touch these lines.